README.md

```diff
@@ -5,15 +5,38 @@ Multihash
 [![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](http://hexdocs.pm/ex_multihash/)
 [![Build Status](https://api.travis-ci.org/multiformats/ex_multihash.svg?branch=master)](https://travis-ci.org/multiformats/ex_multihash)
 [![Inline docs](http://inch-ci.org/github/multiformats/ex_multihash.svg)](http://inch-ci.org/github/multiformats/ex_multihash)
+[![](https://img.shields.io/badge/project-multiformats-blue.svg?style=flat-square)](http://github.com/multiformats/multiformats)
+[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
 
-[Multihash](https://github.com/multiformats/multihash) implementation in Elixir
+> Multihash implementation in Elixir
 
-# How to use it
+This is the [Multihash](https://github.com/multiformats/multihash) implementation in Elixir.
 
-## Encoding
+## Table of Contents
+
+- [Install](#install)
+- [Usage](#usage)
+  - [Encoding](#encoding)
+    - [Examples](#examples)
+    - [Examples](#examples-1)
+    - [Examples](#examples-2)
+  - [Decoding](#decoding)
+    - [Examples](#examples-3)
+    - [Examples](#examples-4)
+- [Maintainers](#maintainers)
+- [Contribute](#contribute)
+- [License](#license)
+
+## Install
+
+TODO
```

Comment:
This should have the following 

To use ex_multihash add to your `mix.exs` file:
```
defp deps do
  [
    {:ex_multihash, "~> 1.0"}
  ]
end
```